### PR TITLE
Fix CLI bug regarding an empty user

### PR DESCRIPTION
**Pull request by Alkarex**
Fix https://github.com/FreshRSS/FreshRSS/issues/2644

### CODE REVIEW

File: cli/_cli.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ function cliInitUser($username) {
 	if (FreshRSS_Context::$user_conf == null) {
 		fail('FreshRSS error: invalid configuration for user: ' . $username . "\n");
 	}
-	new Minz_ModelPdo($username);
+	Minz_Session::_param('currentUser', $username);
```
Comment (Alkarex) [inline cli/_cli.php:40]: We now check later to ensure that the user is set when working with a user database

File: lib/Minz/ModelPdo.php [hunk 2]
```diff
@@ -28,13 +28,13 @@ public function __construct($currentUser = null, $currentPdo = null) {
...
 			$currentUser = Minz_Session::param('currentUser');
 		}
-		if ($currentUser == '') {
-			throw new Minz_PDOConnectionException('Current user must not be empty!', '', Minz_Exception::ERROR);
-		}
 		if ($currentPdo != null) {
 			$this->pdo = $currentPdo;
 			return;
 		}
+		if ($currentUser == '') {
```
Comment (Alkarex) [inline lib/Minz/ModelPdo.php:35]: That check is only relevant when we do not already have a user PDO, so change order of checks

File: cli/_cli.php [hunk 3]
```diff
@@ -37,7 +37,7 @@ function cliInitUser($username) {
 	if (FreshRSS_Context::$user_conf == null) {
 		fail('FreshRSS error: invalid configuration for user: ' . $username . "\n");
 	}
-	new Minz_ModelPdo($username);
```
Comment (Alkarex) [inline cli/_cli.php:40]: This was an old, even less good-looking, work-around

**Review by X-dark — APPROVED** [review]
It does fix #2644 for me. Thanks

**Pull request merged**